components/channel_selector_modal/channel_selector_modal.tsx

```diff
@@ -86,7 +94,7 @@ export default class ChannelSelectorModal extends React.PureComponent {
         }
     }
 
-    handleSubmit = (e) => {
+    handleSubmit = (e: any) => {
```

Comment:
As an aside, @marianunez the `Multiselect` component seems to be sending the generic `T` type here, not an event, so I'm not sure what `e.preventDefault` below is doing...?

Reply:
Agreed, not sure how this is working calling `e.preventDefault()` unless `e` is passed null? It seems that `e` is not used at all besides that in this function